### PR TITLE
Remove migration apps and jobs from master

### PR DESCRIPTION
**Pull request by dkijania**
After hardfork is done we can remove all migration apps, jobs and scripts from our repo

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!help-merge-me

**Comment by dkijania** [comment]
Hello, @dkijania 2 new  created to help you port this change to master. Please follow steps to resolve conflict for each mainline branch. PRs will be created automatically after you push merge to porting branches

  
## compatible 

```
git fetch && git checkout port/_dkijania/remove_berkeley_migration_apps_to_compatible && git merge dkijania/remove_berkeley_migration_apps
```

Resolve conflict and then 

```
git push
```  
    
  
## develop 

```
git fetch && git checkout port/_dkijania/remove_berkeley_migration_apps_to_develop && git merge dkijania/remove_berkeley_migration_apps
```

Resolve conflict and then 

```
git push
```

**Comment by dkijania** [comment]
Porting branch #15798 to resolve conflict with compatible created. Please merge it in order to solve conflict between head of this PR and compatible

**Comment by dkijania** [comment]
Porting branch #15799 to resolve conflict with develop created. Please merge it in order to solve conflict between head of this PR and develop

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-nightly-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by SanabriaRusso — APPROVED** [review]

**Review by svv232 — APPROVED** [review]

File: buildkite/scripts/caqti-upgrade-plus-archive-init-speedup.patch [hunk 1]
```diff
@@ -2795,784 +2795,7 @@ index caa4203939..a2934a6184 100644
    | Error e ->
        [%log fatal]
          ~metadata:[ ("error", `String (Caqti_error.show e)) ]
-diff --git a/src/app/berkeley_account_tables/berkeley_account_tables.ml b/src/app/berkeley_account_tables/berkeley_account_tables.ml
```
Comment (georgeee) [inline buildkite/scripts/caqti-upgrade-plus-archive-init-speedup.patch:2798]: This patch was useful (it upgraded caqti which allowed us to optimize resource usage which was critical for replayer, but might appear to be critical for something else later)

Let's create a follow-up PR/issue to have this patched upstreamed

**Review by georgeee — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

File: buildkite/scripts/caqti-upgrade-plus-archive-init-speedup.patch (hunk 1 above)
Comment (dkijania) [reply buildkite/scripts/caqti-upgrade-plus-archive-init-speedup.patch:2798]: I just removed diff for apps i removed. I will create follow up PR for applying this patch

**Pull request merged**